app/models/manageiq/providers/microsoft/infra_manager/powershell.rb

```diff
@@ -93,7 +95,11 @@ def run_powershell_script(script)
 
       _result, timings = Benchmark.realtime_block(:execution) do
         with_provider_connection do |connection|
-          results = connection.create_executor { |exec| exec.run_powershell_script(script) }
+          script_string = ""
+          File.open(script, "r") do |file|
+            script_string << file.read
+          end
```

Comment:
@djberg96 these are obviously better.  I'll update in the AM - testing another config at the moment.  Thx.